src/plugins/exportFile/test/exportFile.e2e.js

```diff
@@ -94,11 +94,9 @@ describe('exportFile', () => {
 
       const result = plugin._createBlob(formatter);
 
-      if (!Handsontable.helper.isIE9()) {
```

Comment:
`isIE9` was reverted. Shouldn't this condition be reverted as well?

Reply:
As we planned to drop support for IE9 in the next months I decided to not revert "happy test" for IE9. Moreover, I suspect that not only that test doesn't pass on IE9.